ppSchema.sql

```diff
@@ -0,0 +1,17 @@
+-- create tables
+-- A ping-pong tournament app, with `users` and `games`
+
+create table players (
+    id serial primary key,
+    name text,
+    age integer,
+    country text
+);
+
+create table games (
+    id serial primary key,
+    date_time timestamp,
+    country1 text,
```

Comment:
In this case, you might want to have two foreign keys to the `players` table, that way you can just derive which countries are playing each other.